packages/editor/src/components/post-author/index.js

```diff
@@ -9,13 +9,14 @@ import { store as coreStore } from '@wordpress/core-data';
  */
 import PostAuthorCombobox from './combobox';
 import PostAuthorSelect from './select';
+import { AUTHORS_QUERY } from './constants';
 
 const minimumUsersForCombobox = 25;
 
 function PostAuthor() {
 	const showCombobox = useSelect( ( select ) => {
-		// Not using `getUsers()` because it requires `list_users` capability.
-		const authors = select( coreStore ).getAuthors();
```

Comment:
Should we deprecate this selector and remove the unstable one?

Reply:
Yes, that's the plan for the next PR :)